packages/components/app/styles/components/application-state.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APPLICATION-STATE COMPONENT
+//
+$hds-application-state-padding: 12px 0;
+
+.hds-application-state {
+  align-self: center;
+  width: 20rem;
+  max-width: 100%;
+  margin: 0 auto;
+}
+
+.hds-application-state__header,
+.hds-application-state__body,
+.hds-application-state__footer {
+  display: flex;
+}
+
+.hds-application-state__header {
+  display: grid;
+  grid-row-gap: 0;
+  grid-column-gap: 0;
+  color: var(--token-color-foreground-primary);
+
+  .hds-application-state--error & {
+    grid-template-rows: repeat(2, auto);
+    grid-template-columns: 28px auto;
```

Comment:
If you used gap instead to add space between the icon and text then you could change this so the 1st column "shrink-wraps" the SVG.

IE:
`grid-template-columns: 28px auto;`
=>
`grid-template-columns: min-content auto;`